services/api/service.go

```diff
@@ -457,6 +477,54 @@ func (api *RelayAPI) startValidatorRegistrationDBProcessor() {
 	}
 }
 
+func (api *RelayAPI) processPayloadAttributes(payloadAttributes beaconclient.PayloadAttributesData) {
+	// only for builder-api and if using see subscriptions instead of querying for payload attributes
+	if api.opts.BlockBuilderAPI && api.ffEnableSSEPayloadAttributes {
```

Comment:
rather than nesting, return early

```suggestion
	if !api.opts.BlockBuilderAPI {
		return
	}
```

not sure `api.ffEnableSSEPayloadAttributes` needs to be checked, because it'll only be called if enabled anyway?